style.css

```diff
@@ -1,26 +1,45 @@
-.follow-nature li {
-  position: relative;
+.follow-nature ul {
+    overflow: visible;
+}
 
+.follow-nature li {
+    position: relative;
 }
 
-a::after {
-  content: "RSS feed";
-  position: absolute;
-  background-color: #000;
-  padding: 10px 20px;
-  color: #fff;
-  border-radius: 15px;
-  top: 30px;
-  left: 50%;
-  width:auto;
-  font-family: monospace;
-  z-index: 99;
-  display: none;
-  transform: translateX(-50%);
+.follow-nature a::after {
+    position: absolute;
+    background-color: #000;
+    padding: 10px 20px;
+    color: #fff;
+    border-radius: 15px;
+    top: 35px;
+    left: 50%;
+    font-family: monospace;
+    display: none;
+    transform: translateX(-50%);
 }
 
 .follow-nature a:hover::after {
-  display: block;
-  color: #fff;
-  position: absolute;
+    display: block;
+    text-indent: 0;

```

Comment:
the stylesheet that the code test came with had `text-indent: -999em` applied to the `.follow-nature ul li a` which was quite sneaky and probably [purposefully] confused a lot of people doing the code test. This cancels that out to ensure that we an see the text (otherwise the text is miles off the page and you cant see it)